src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -14,6 +14,13 @@ import { replaceDomainsInData } from '../../../utilities/environment/stagingDoma
 
 import MegaMenu from '@department-of-veterans-affairs/formation/MegaMenu';
 
+// The MegaMenu data is generated out of vagov-content, rather than a static JSON file.
+// During the build, the resultant link data is first written into temporary storage,
+// and then required by Webpack to be made available under __MEGAMENU_CONFIG__.
+
+// eslint-disable-next-line no-undef
```

Comment:
I was thinking of leaving this out of the eslint config, because `__BUILDTYPE__` and `__API__` aren't defined there either. Should we define all three?

Reply:
Yeah, that seems reasonable.

Reply:
@jbalboni Added them in https://github.com/department-of-veterans-affairs/vets-website/pull/9368/commits/f3ced1be7d68b5d04a3dfa3206428ad0c1f4ed3d. I like that change, but I'm a little concerned about allowing access to `__BUILDTYPE__` again.